### PR TITLE
[Enhancement] Added Task Reminder feature through local notification

### PR DESCRIPTION
**Pull request by Interstellar07**
This PR is in reference to issue #63 

I have implemented a local notification feature to remind users of their task deadlines in case they miss them and It even works even though the app is background state making sure users are up-to-date with their tasks and deadlines.

Images Attached Below:
image

PR passes all the guidelines as set by the organization and maintainer.

### CODE REVIEW

**Review by NishantSinghal19 — APPROVED** [review]
Nice Code ! It is working properly as well as maintaining all the other condition of app

**Comment by Interstellar07** [comment]
> Nice Code ! It is working properly as well as maintaining all the other condition of app

Thank you so much for these appreciation words! It means a lot :)

File: lib/widgets/buildTasks.dart [hunk 1]
```diff
@@ -61,138 +62,161 @@ class _TasksBuilderState extends State<TasksBuilder> {
...
-                        label: "DELETE",
-                        backgroundColor: Colors.red,
-                      ),
-                    ],
-                  ),
-                  child: Card(
-                    color:
-                        AppSettings.isDarkMode ? Palette.kToDark : Colors.white,
-                    child: InkWell(
-                      customBorder: RoundedRectangleBorder(
```
Comment (Pavel401) [inline lib/widgets/buildTasks.dart:153]: Hey, maybe you have removed this part of the code which is fixed for this issue https://github.com/CCExtractor/taskwarrior-flutter/issues/65

 "customBorder: RoundedRectangleBorder(
                      borderRadius: BorderRadius.circular(10),
                    ),"

File: lib/widgets/buildTasks.dart [hunk 2]
```diff
@@ -61,138 +62,161 @@ class _TasksBuilderState extends State<TasksBuilder> {
...
+                  darkmode: AppSettings.isDarkMode,
+                ),
+              ),
+            ),
+          )
               : Card(
-                  color:
-                      AppSettings.isDarkMode ? Palette.kToDark : Colors.white,
-                  child: InkWell(
-                    customBorder: RoundedRectangleBorder(
```
Comment (Pavel401) [inline lib/widgets/buildTasks.dart:177]: same here

**Review by Pavel401 — COMMENTED** [review]
Please check

File: lib/widgets/buildTasks.dart (hunk 2 above)
Comment (Pavel401) [reply lib/widgets/buildTasks.dart:177]: @Interstellar07 @NishantSinghal19 please check

Comment (Interstellar07) [reply lib/widgets/buildTasks.dart:177]: will be resolved if holds true, Thanks!

**Pull request merged**